x-pack/solutions/security/plugins/security_solution/docs/rfcs/detection_response/customized_rule_alert_telemetry.md

```diff
@@ -0,0 +1,273 @@
+- Once we add it there's (basically) no going back
+  - Not necessarily a con but something to consider in terms of longevity 
+
+### Complexity & Cost
+
+This effort would involve:
+ -  adding a new field to our rule schemas
+ -  updating all dependent schemas
+ -  adding the logic to update this field all the same places we update `rule_source`
+ -  writing the new field to the alert documents during rule execution
+ -  writing tests to cover the new field and resulting logic additions
+
+This is all in addition to the then-unblocked telemetry work that could take place once this was implemented.
+
+---
+
+## Appendix
+
+### Defining functional rule fields
+
+| Rule fields                           | Functional field? |
+|---------------------------------------|-------------------|
+| `name`                                | no                |
+| `description`                         | no                |
+| `risk_score`                          | yes               |
+| `severity`                            | yes               |
+| `rule_name_override`                  | yes               |
+| `timestamp_override`                  | yes               |
+| `timestamp_override_fallback_disabled`| yes               |
+| `timeline_id`                         | no                |
+| `timeline_title`                      | no                |
+| `license`                             | no                |
+| `note`                                | no                |
+| `building_block_type`                 | no                |
+| `investigation_fields`                | no                |
+| `version`                             | no                |
+| `tags`                                | no                |
+| `risk_score_mapping`                  | yes               |
+| `severity_mapping`                    | yes               |
+| `interval`                            | yes               |
+| `from`                                | yes               |
+| `to`                                  | yes               |
+| `author`                              | no                |
+| `false_positives`                     | no                |
+| `references`                          | no                |
+| `max_signals`                         | yes               |
+| `threat`                              | no                |
+| `setup`                               | no                |
+| `related_integrations`                | no                |
+| `required_fields`                     | no                |
+| `query`                               | yes               |
+| `type`                                | yes               |
+| `language`                            | yes               |
+| `index`                               | yes               |
+| `data_view_id`                        | yes               |
+| `filters`                             | yes               |
+| `event_category_override`             | yes               |
```

Comment:
```suggestion
| `event_category_override`             | no               |
```

? Same thing here. Does this change just a field in the final alert, versus the trigger conditions?